src/wasm-interpreter.h

```diff
@@ -3609,9 +3609,11 @@ template<typename GlobalManager, typename SubType> class ModuleInstanceBase {
     return flow.values;
   }
 
+  // The maximum call stack depth to evaluate into.
+  static const Index maxDepth = 250;
```

Comment:
Should we prefer `constexpr` over `const` these days?

Reply:
(I don't have an opinion on const/constexpr myself...)

Reply:
I heard that `constexpr` can be win because it means that compiler can assume to really is const.  (as opposed to `const` which can be cast away with `const_cast` and modified.. not sure that applies to basic types like..)